src/pages/profile/components/ProfileCarousel.tsx

```diff
@@ -32,9 +35,24 @@ const ProfileCarousel = ({ tabItems, fullName }: ProfileCarouselProps) => {
       {Object.entries(tabItems).map(([label, tabItem], index) => {
         switch (label) {
           case MEDITATION:
-            return (
+            return tabItem.data && tabItem.data.length > 0 ? (
               <ProfileCarouselItem key={index}>
-                {tabItem.value}
+                {tabItem.data &&
+                  tabItem.data.map((post, index) => (
+                    <PostPreview
+                      key={index}
+                      post={post as Post}
+                      totalLikes={post.likes.length}
+                      totalComments={post.comments.length}
+                      noneProfile={true}
```

Comment:
<img width="627" alt="스크린샷 2023-09-25 오후 12 19 45" src="https://github.com/prgrms-fe-devcourse/FEDC4_NIRVANA_Gidong/assets/69716992/063fddf7-b051-4e4a-b5af-58b5c7b4bcae">

```suggestion
                  tabItem.data.map((post, index) => {
                    const { likes, comments } = post as Post;
                    return (
                      <PostPreview
                        key={index}
                        post={post as Post}
                        totalLikes={likes.length}
                        totalComments={comments.length}
                        noneProfile={true}
```

구조분해할당으로 likes와 comments를 넘겨주면 타입 오류가 사라집니다!